Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -496,7 +498,8 @@ end OrderedCommMonoid
 
 section OrderedCommGroup
 
-variable {hα : OrderedCommGroup α} {hβ : OrderedCommGroup β}
+variable [CommGroup α] [PartialOrder α] [IsOrderedMonoid α]
```

Comment:
Do you understand why these were implicit variables instead of typeclasses? In the previous section, you have respected this choice, but here you have changed.